messenger/make.py

```diff
@@ -68,45 +67,34 @@ def build_octave():
 
 
 def build_matlab(static=False):
-    """build the messenger mex for MATLAB
-    
+    """
+    Build the messenger mex for MATLAB
+
+    Parameters
+    ============
     static : bool
         Determines if the zmq library has been statically linked.
         If so, it will append the command line option -DZMQ_STATIC
         when compiling the mex so it matches libzmq.
     """
-    cfg = get_config()
-    # To deal with spaces, remove quotes now, and add
-    # to the full commands themselves.
-    matlab_bin = cfg['matlab_bin'].strip('"')
-    # Get the extension
-    extcmd = '"' + os.path.join(matlab_bin, "mexext") + '"'
-    extension = subprocess.check_output(extcmd, shell=True)
-    extension = extension.decode('utf-8').rstrip('\r\n')
+    matlab_bin = settings.get_matlab_bin()
+    cfg, host = ConfigParser(), platform.system()
+    cfg.read('config.ini')
+    libzmq = {
+        'zmq_lib': os.path.normpath(cfg.get(host, 'ZMQ_LIB')),
+        'zmq_inc': os.path.normpath(cfg.get(host, 'ZMQ_INC')),
+    }
+
+    extcmd     = '%s' % os.path.join(matlab_bin, "mexext")
```

Comment:
There used to be quotes around the command here. I'm pretty sure they're still needed so you should add them back -- if not, you can drop the string interpolation altogether.